src/runtime/pkg/katautils/config-settings.go.in

```diff
@@ -117,3 +117,5 @@ const defaultPCIeSwitchPort = 0
 
 const defaultRemoteHypervisorSocket = "/run/peerpod/hypervisor.sock"
 const defaultRemoteHypervisorTimeout = 600
+
+const defaultSnpCertsPath = "/opt/kata/share/kata-qemu-snp-experimental/certs"
```

Comment:
Who'll be deploying this?

If this is deployed by kata-deploy, then I'd say we're missing artefacts / logic on the kata-deploy side.

If this is not deployed by kata-deploy, ten /opt/kata is not the rigt place to consume the file from.

Reply:
This will not be deployed by kata-deploy. But, we wanted to have a default path for the file containing the certificates.  Do you have any suggestion? What about /var/snp/certs or /opt/snp/certs ?

Reply:
It's more about what the tool used to deploy those certs will have access to.
Anything but mixing with the content from kata-deploy would be good.

While here, can you expand on how those certs will be deployed?

Reply:
One could use the tool [snphost](https://github.com/virtee/snphost/blob/main/docs/snphost.1.adoc) on the SNP host to create the cert-chain including VCEK/VLEK certificates and provide the path of the cert-chain in the config file (i.e. /opt/kata/share/defaults/kata-containers/configuration-qemu-snp.toml ).

Reply:
Makes sense.  I assume the path chosen for you in the end is the default path used by the tool.